th2_common/schema/factory/abstract_common_factory.py

```diff
@@ -99,7 +99,7 @@ def message_raw_batch_router(self) -> MessageRouter:
         return self._message_raw_batch_router
 
     @property
-    def event_batch_router(self) -> MessageRouter:
+    def event_batch_router(self) -> EventBatchRouter:
```

Comment:
Why should we replace interface with implementation?

Reply:
You are right. Will be reverted